.gitlab/package_build/rpm.yml

```diff
@@ -114,9 +114,8 @@ agent_rpm-arm64-a6:
 agent_rpm-arm64-a7:
   extends: .agent_build_common_rpm
   rules:
-    - !reference [.on_all_builds]
-    - !reference [.on_packaging_change]
-    - !reference [.on_go-version_change]
+    - !reference [.except_mergequeue]
+    - when: on_success
```

Comment:
I suppose this is because otherwise the tests job complain about a dependent job not being part of the pipeline? I'm not quite sure this we want to systematically build arm64 agents on PRs pipelines for now

Reply:
No strong opinion on my side between this & just adding the `new-e2e-installer` trigger. I added these to be consistent between arm64 & amd64.

Reply:
I think the arm64 build will be used only when your tests need to run on PR. Thus, this is a bit sad to add two build jobs that will be used not that often. 
Do you think you could make it work with the old rules and simply adding `.on_installer_or_e2e_changes_or_manual` to the existing rules? You will probably need to do the same with deploy jobs.

With that you should have the ARM build when your test need to be executed and a manual job otherwise